Resources/Private/Fusion/Core/FormElement.fusion

```diff
@@ -7,17 +7,17 @@ prototype(Neos.Form.FusionRenderer:FormElement) < prototype(Neos.Fusion:Array) {
     label = Neos.Form.FusionRenderer:FormElementLabel
     fieldContainer = Neos.Form.FusionRenderer:FormElementFieldContainer
 
-    @process.renderCallbacks = Neos.Form.FusionRenderer:RenderCallbacks {
-        formRuntime = ${formRuntime}
-        formElement = ${element}
-    }
-
     @process.wrap = Neos.Fusion:Tag {
         tagName = 'div'
         attributes.class = ${hasValidationErrors ? 'clearfix error' : 'clearfix'}
         content = ${value}
     }
 
+    @process.renderCallbacks = Neos.Form.FusionRenderer:RenderCallbacks {
```

Comment:
Isn't using the `@position` safer than moving the code?

Reply:
Done